samples/snippets/visualbasic/VS_Snippets_CLR/x509certificate2/vb/program.vb

```diff
@@ -226,7 +226,7 @@ Class Program
                 Directory.CreateDirectory(decrFolder)
                 '<Snippet10>
                 ' Use RSACryptoServiceProvider
-                ' to decrypt the Rijndael key.
+                ' to decrypt the AES key.

```

Comment:
Since the file's already being edited: AesManaged -> Aes; and delete the line that sets BlockSize.  (AES has only one legal BlockSize).